ApiCheck/Loader/AssemblyLoader.cs

```diff
@@ -15,21 +15,21 @@ public void Dispose()
     {
       AppDomain.CurrentDomain.ReflectionOnlyAssemblyResolve -= CurrentDomainOnReflectionOnlyAssemblyResolve;
     }
-    
+
     private Assembly CurrentDomainOnReflectionOnlyAssemblyResolve(object sender, ResolveEventArgs args)
     {
       AssemblyName assemblyName = new AssemblyName(args.Name);
       string path = Path.Combine(Path.GetDirectoryName(args.RequestingAssembly.Location), assemblyName.Name + ".dll");
       if (File.Exists(path))
       {
-        return Assembly.ReflectionOnlyLoadFrom(path);
+        return Assembly.LoadFile(path);
       }
       return Assembly.ReflectionOnlyLoad(AppDomain.CurrentDomain.ApplyPolicy(args.Name));
     }
 
     public Assembly ReflectionOnlyLoad(string path)
```

Comment:
Method name does not match the implementation anymore.

Reply:
Same as above.